plasmapy/formulary/parameters.py

```diff
@@ -694,6 +702,15 @@ def Hall_parameter(n,
     Returns
     -------
     astropy.units.quantity.Quantity
+
+    Examples
+    --------
+    >>> from astropy import units as u
+    >>> Hall_parameter(1e10 * u.m**-3, 2.8e3 * u.eV, 2.3 * u.T, 'He-4 +1')
+    <Quantity 7.26446755e+16>
+    >>> Hall_parameter(1e10 * u.m**-3, 5.8e3 * u.eV, 2.3 * u.T, 'He-4 +1')
+    <Quantity 3.25995498e+17>
```

Comment:
Tests seem to output the following value here:
```suggestion
    <Quantity 2.11158408e+17>
```
I may have messed the particle up while adding whitespace, didn't you have a proton or some such here before?

Anyway, it should be fine to keep it as it is :)

Reply:
I just pushed  the change